src/transport/AdminPairingTable.cpp

```diff
@@ -28,6 +28,18 @@ namespace chip {
 
 namespace Transport {
 
+CHIP_ERROR AdminPairingInfo::SetFabricLabel(uint8_t * fabricLabel)
```

Comment:
Need to have `const uint8_t *` for input argument, since it doesn't get mutated.